src/test/scala/edu/ie3/osmogrid/io/input/OsmSourceSpec.scala

```diff
@@ -0,0 +1,116 @@
+/*
```

Comment:
This looks like an old copy of what is now `OsmSourceIT`, same as with the `inputTest.pbf`, which is now called `Witten-Stockum.pbf`, located in test resources. Maybe this results from a flawed merge :)